### ISSUE TITLE
gp preview requires a click before it opens Simple Browser

### ISSUE THREAD

**Issue by apolopena** [gitpod-io/gitpod#7904]
### Bug description

`gp preview` fires off and nothing happens in the VSCode cloud IDE until the user clicks in the view port of the getting started tab.

### Steps to reproduce

1. Create a new workspace
   - https://gitpod.io/#/https://github.com/apolopena/gp-react-on-rails-starter
2. Wait for the build
3. Observe the output in the rightmost terminal
4. When the output in the rightmost terminal reads:
```log
Opening preview when server is ready [DONE]
Enjoy your coding experience 👩‍💻👨‍💻
```
The preview browser should automatically open but nothing happens until you click in the viewport.

### Workspace affected

apolopena-gpreactonrails-j01ihal2nst.ws-us29

### Expected behavior

Simple Browser should open without user interaction when a `gp preview` call is made.

### Example repository

https://github.com/apolopena/gp-react-on-rails-starter

### Anything else?

Some context regarding the example repository and how gp preview is being used:

1. `gp sync-await` and `gp sync-done` are being used to open the preview at the right time.
2. `gp preview command is being called through a bash alias function call to https://github.com/apolopena/gp-react-on-rails-starter/blob/main/.gp/bash/open-preview.sh

**Comment by shaal** [comment]
I have this exact issue.
I use bash for gp preview.
I can replicate it by opening https://github.com/shaal/DrupalPod in gitpod.

**Comment by axonasif** [comment]
I'm unable to reproduce this on my system.
OS: Macintosh 12.2
Browser: Chrome Version 97.0.4692.99 (Official Build) (arm64)

I just opened your link and did no input either from my trackpad or keyboard, I even hid the mouse pointer in a corner before the workspace launched.

**Comment by apolopena** [comment]
The bug appears to be intermittant.

**Comment by axonasif** [comment]
Could be.

**Comment by shaal** [comment]
I was about to tell you, try my repo, it's ALWAYS like that.
Then I tried it myself, and preview just worked... without clicking the mouse.

The reason I didn't open this issue before, is that I couldn't find a simple enough repo to replicate the problem with.

**Comment by shaal** [comment]
@axonasif @apolopena 
I see this issue today, try this link:
https://gitpod.io/#https://github.com/shaal/DrupalPod

Do not click with the mouse on anything.
See that the terminal finish running all the commands.
Click with the mouse in the terminal - and you'll see the preview displayed.

**Comment by apolopena** [comment]
@shaal 
I just tried your repo and could not replicate. it This is an elusive bug indeed.
Nice work on Drupalpod by the way.

**Comment by shaal** [comment]
@apolopena Thank you :) yes, I cannot find how to replicate the bug in a consistent way.

**Comment by axonasif** [comment]
Could you guys take a look at your console logs when this happens, maybe something interesting there?

**Comment by shaal** [comment]
@axonasif console has lots of warnings and errors, but that's normal as far as I know.

![image](https://user-images.githubusercontent.com/22901/151922454-7cea6809-dda9-4522-a2da-ee6e3e2144e3.png)

After clicking in terminal these lines are added in `console.log`:
```
Error with Permissions-Policy header: Unrecognized feature: 'interest-cohort'.
shaal-drupalpod-dgtnd229dih.ws-us29.gitpod.io/:1 [Intervention] Images loaded lazily and replaced with placeholders. Load events are deferred. See https://go.microsoft.com/fwlink/?linkid=2048113
```

**Comment by apolopena** [comment]
I think it might have to do with the entire browser window not having focus but being on screen like in the case where there are mulitple monitors involved. I will investigate this further.

**Comment by apolopena** [comment]
Yeah my tests show this happens when the entire browser window does not have focus. I updated the bug, reproduction steps, and a respository that isolates the bug to it's simplest implmentation. Please give it a whirl.

**Comment by loujaybee** [comment]
Thanks for raising @apolopena, we'll take a look into this one 🙏 

Currently scheduled for investigation / fix with the IDE team :)

**Comment by shaal** [comment]
Please add the label `meta: never-stale` to this issue, this issue is still a problem.
